test/suite/README

```diff
@@ -0,0 +1,13 @@
+The test/suite directory includes a collection of Python unit tests 
```

Comment:
Thanks for adding this - as described above, I'd like to change the bucketing so it won't miss new tests.

I'd meant to ask for a README in the csuite directory - describing what we'd done there.

Reply:
The README for csuite would be covered by WT-4454.